package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "TODO",
+  "name": "IOAppCIE",
```

```suggestion
  "name": "io-app-cie",
```
kebab case is preferred to name the project in `package.json`